File: java/src/test/java/mx/iteso/MilkIceCreamStoreTest.java
package mx.iteso;

import static org.junit.Assert.assertEquals;
import org.junit.Test;
import mx.iteso.Cone;
import mx.iteso.stores.MilkIceCreamStore;

public class MilkIceCreamStoreTest {
    @Test
    public void milkIceCreamStoreGuavaTest() {
        Cone cone = new MilkIceCreamStore().prepareCone("Guava");
        assertEquals(cone.getIceCream().getFlavor(), "Leche -> Guayaba");
    }

    @Test
    public void milkIceCreamStoreMangoTest() {
        Cone cone = new MilkIceCreamStore().prepareCone("Mango");
        assertEquals(cone.getIceCream().getFlavor(), "Leche -> Mango");
    }

    @Test
    public void milkIceCreamStorePineappleTest() {
        Cone cone = new MilkIceCreamStore().prepareCone("Pineapple");
        assertEquals(cone.getIceCream().getFlavor(), "Leche -> Piña");
    }

    @Test
    public void milkIceCreamStoreStrawberryTest() {
        Cone cone = new MilkIceCreamStore().prepareCone("Strawberry");
        assertEquals(cone.getIceCream().getFlavor(), "Leche -> Fresa");
    }
}

File: python/src/unittest/python/MilkIceCreamStoreTest.py
from mockito import mock, verify, when
import unittest
from store.MilkIceCreamStore import *


class MilkIceCreamStoreTest(unittest.TestCase):

    def setUp(self):
        self.store = MilkIceCreamStore()

    def guavaMilkIceCreamTest(self):
        cone = self.store.prepareCone("Guava")
        self.assertEquals(cone.getIceCream().getFlavor(), "Leche -> Guayaba")

    def mangoMilkIceCreamTest(self):
        cone = self.store.prepareCone("Mango")
        self.assertEquals(cone.getIceCream().getFlavor(), "Leche -> Mango")

    def pineappleMilkIceCreamTest(self):
        cone = self.store.prepareCone("Pineapple")
        self.assertEquals(cone.getIceCream().getFlavor(), "Leche -> Piña")

    def strawberryMilkIceCreamTest(self):
        cone = self.store.prepareCone("Strawberry")
        self.assertEquals(cone.getIceCream().getFlavor(), "Leche -> Fresa")